frontend/lib/ui/ProgressList/ProgressList.module.css

```diff
@@ -10,28 +10,41 @@
     display: flex;
     align-items: center;
     gap: 0.5rem;
+    padding: 0.625rem 0 0.625rem 0.75rem;
 
-    &:hover {
+    :not(&:global(.ruler)):not(&:global(.disabled)):not(&:global(.active))&:hover {
```

Comment:
i find this a bit hard to grok.
Imho it's nicer to have default styles and override them in variants.

Reply:
I have improved this